apps/robots-generator/src/components/ExistingRobots/ExistingRobots.tsx

```diff
@@ -0,0 +1,158 @@
+import { Box, Button, Snackbar, Stack, Typography } from "@mui/material";
+import Alert from "@mui/material/Alert";
+import Checkbox from "@mui/material/Checkbox";
+import FormControlLabel from "@mui/material/FormControlLabel";
+import FormGroup from "@mui/material/FormGroup";
+import { useState } from "react";
+
+import Input from "@/robots-generator/components/Input";
+import StepperNav from "@/robots-generator/components/StepperNav";
+import { useGlobalState } from "@/robots-generator/context/GlobalContext";
+import { validateUrl } from "@/robots-generator/utils/urls";
+
+interface ExistingRobotsProps {
+  // eslint-disable-next-line no-unused-vars
```

Comment:
We may need to adjust eslint config not to throw an error on unused variables inside type definitions